src/main/java/com/agnieszkapawska/flashcards/domain/facades/FlashcardFacade.java

```diff
@@ -87,4 +88,16 @@ public FlashcardSaveResponseDto updateFlashcard(FlashcardDto flashcardDto, Long
         });
         return uselessQuestionTags;
     }
+
+    public List<FlashcardGetDto> getAllFlashcards(Optional<String> searchPhrase) {
```

Comment:
So all flashcards or only those who contain searchPhrase? `getFlashcards` would be enough.